library/module_utils/oneview.py

```diff
@@ -330,6 +330,7 @@ class OneViewModule(object):
     MSG_ALREADY_PRESENT = 'Resource is already present.'
     MSG_ALREADY_ABSENT = 'Resource is already absent.'
     MSG_DIFF_AT_KEY = 'Difference found at key \'{0}\'. '
+    MSG_MANDATORY_FIELD_MISSING = 'Mandatory field was not informed: name'
```

Comment:
The message wording can be changed something like "Missing mandatory field : name'

Reply:
As its a base class, kept all the generic messages here used by resource classes. Yes I will update the text

Reply:
Updated the text, please have a look.